docs/website/layouts/index.html

```diff
@@ -234,17 +234,19 @@ <h2>Declarative Policy</h2>
                                         </pre>
                                 </div>
                                 <div class="highlight">
```

Comment:
If everything is done properly, each new playground link should appear four times:

1. code snippet on deskop
2. "open in playground" button on desktop
1. code snippet on mobile
2. "open in playground" button on mobile

it's a copy-paste accident waiting to happen 😑